app/controllers/miq_request_controller.rb

```diff
@@ -400,6 +400,17 @@ def states_for_checkboxes_i18n
     end
   end
 
+  def user_options(params)
+    set_defaults = prov_set_default_options
+    user_defaults = {
+      :reason_text    => params["reasonText"].nil? ? set_defaults[:reason_text] : params["reasonText"],
+      :applied_states => params["approvalStateCheckboxes"].nil? ? set_defaults[:applied_states] : params["approvalStateCheckboxes"],
+      :type_choice    => params["types"].nil? ? set_defaults[:type_choice] : params["types"],
+      :user_choice    => params["selectedUser"].nil? ? set_defaults[:user_choice] : params["selectedUser"],
+      :time_period    => params["selectedPeriod"].nil? ? set_defaults[:time_period] : params["selectedPeriod"],
```

Comment:
similar suggestion for all 4 of the above, unless one of them is expected to return `false`